src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/RuntimeBroker.cs

```diff
@@ -451,7 +451,7 @@ public async Task RemoveAccountAsync(ApplicationConfiguration appConfig, IAccoun
             ICacheSessionManager cacheSessionManager,
             IInstanceDiscoveryManager instanceDiscoveryManager)
         {
-            if (!_wamOptions.ListWindowsWorkAndSchoolAccounts)
+            if (!_wamOptions.ListOperatingSystemAccounts)
```

Comment:
```suggestion
            if (!_wamOptions.ListWindowsWorkAndSchoolAccounts)
```

leave as is, if not every log needs to be changed. see the next few lines and you see two different ways of referencing. Why change this?

Reply:
I want to remove specific operating system from it. What if we add capability to another OS?